services/discovery/template.yaml

```diff
@@ -16,8 +16,10 @@ Resources:
   DiscoveryFunction:
     Type: AWS::Serverless::Function 
     Properties:
-      CodeUri: src/
-      Handler: app.handler
+      CodeUri:
```

Comment:
And here we reference the lambda code dynamically in whichever region the stack is being deployed